conductr_cli/resolvers/bintray_resolver.py

```diff
@@ -21,9 +21,23 @@
 BINTRAY_CONDUCTR_CORE_PACKAGE_NAME = 'ConductR-Universal'
 BINTRAY_CONDUCTR_AGENT_PACKAGE_NAME = 'ConductR-Agent-Universal'
 
+SCHEME_SUPPORTED_BY_OTHER_RESOLVERS = ['http', 'file', 'ftp', 's3']
```

Comment:
Isn't this going to present a maintenance issue... isn't it better to concern oneself with just the URI schemes that the resolver supports?